.github/workflows/app_build.yaml

```diff
@@ -0,0 +1,49 @@
+name: NoPorts Desktop Builds
+
+on:
+  push:
+    branches:
+      - trunk
```

Comment:
This should be on tag & optionally on workflow_dispatch. We shouldn't rebuild the app everytime something is pushed to trunk